src/oralb_ble/parser.py

```diff
@@ -287,3 +293,26 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         self.update_binary_sensor(
             str(OralBBinarySensor.BRUSHING), bool(state == 3), None, "Brushing"
         )
+
+    async def async_poll(self, ble_device: BLEDevice) -> SensorUpdate:
+        """
+        Poll the device to retrieve any values we can't get from passive listening.
+        """
+        client = await establish_connection(BleakClient, ble_device, ble_device.address)
```

Comment:
We should use the client with service cash from bleak retry connector

Reply:
Okay - I changed over to BleakClientWithServiceCache, but that object doesn't really look like it does anything more, it states "No longer used since bleak 0.17+ has service caching built-in." under the set_cached_services() function

Reply:
That docstring is a bit stale. I need to fix it. sorry about that